pyproject.toml

```diff
@@ -100,20 +100,20 @@ doc = [
 ]
 test = [
     "pytest",
-    "pytest-cov",
+    "coverage",
 ]
 
 [tool.hatch.version]
 source = "vcs"
 
 [tool.coverage.run]
-source = ["pertpy"]
+source_pkgs = ["pertpy"]
 omit = [
     "**/test_*.py",
 ]
 
 [tool.pytest.ini_options]
-testpaths = "pertpy/tests"
+testpaths = "tests"
```

Comment:
This was independently broken. With my change, plain `pytest` will work as expected and try to collect tests only in the `tests` directory